.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,595 @@
+					// color for the "default" this variant, based on both design in
+					// Zeplin as well as design trends in Figma. Please make any corrections.
+					button-default:
+						(
+							color: var(--global-neutral-10),
+							background-color: var(--global-white),
+						),
+					button-default-hover: (
+						background-color: var(--global-neutral-1),
+					),
+					// DESIGN ASSUMPTION: Date does not require component styling.
+					// DESIGN ASSUMPTION: Grid does not require component styling.
+					carousel:
+						(
+							--slides: 1,
+						),
+					carousel-actions: (
+						display: none,
+						padding-left: var(--global-spacing-3),
+						padding-right: var(--global-spacing-3),
+						//
+						// The width should place the action buttons near the edge of the carousel responsively
+						//
+						// width is derived from adding
+						//   --layout-content-max-width
+						//   2x button widths (--global-font-size-9)
+						//   2x button margins (--global-font-size-5)
+						// then dividing those by
+						//   --layout-max-width
+						// using the formula: (48 + 32 + 1216 + 32 + 48) / 1440
+						//
+						width:
+							calc(calc(var(--layout-content-max-width) + 160) / var(--layout-max-width) * 100%),
+					),
+					carousel-button: (
+						box-shadow: var(--global-box-shadow-1),
+						border-radius: var(--global-spacing-1),
+						background-color: var(--global-white),
+						height: var(--global-spacing-7),
+						width: var(--global-spacing-7),
+					),
+					carousel-track: (
+						gap: var(--global-spacing-2),
+						width: var(--layout-content-scale-width),
+					),
+					heading: (
+						font-weight: var(--global-font-weight-7),
+					),
+					icon: (
+						fill: var(--global-neutral-8),
+						height: var(--global-spacing-4),
+						width: var(--global-spacing-4),
+					),
+					input: (
+						align-items: flex-start,
+						display: flex,
+						flex-direction: column,
+						gap: var(--global-spacing-1),
+					),
+					// DESIGN QUESTION:
+					// There are multiple paddings for input in Figma.
+					// Is this the correct one for a standard, default text input?
+					input-input:
+						(
+							border: 1px solid var(--global-neutral-5),
+							border-radius: var(--global-border-radius-1),
+							padding: var(--global-spacing-1) var(--global-spacing-2),
+						),
+					// DESIGN ASSUMPTION:
+					// This focused input outline is a close but not exact match to the
+					// design in Figma. In figma, the outline inner border is 1px with
+					// outer border of 2px. This 3px double outline results in a 2px
+					// inner border with 1px outer border. Was there another
+					input-input-focus:
+						(
+							outline: 3px double var(--global-neutral-5),
+						),
+					input-label: (
+						display: block,
+						font-weight: var(--global-font-weight-7),
+					),
+					// DESIGN QUESTION:
+					// The input error border color in Figma is not one of the red
```

Comment:
is this a todo? if so, can you add the new red tokens and remove this

Reply:
Updated. Comment removed.